docs/tutorials/saving_metrics.rst

```diff
@@ -0,0 +1,79 @@
+.. _saving_metrics:
+
+###############
+Saving Metrics
+###############
+
+Saving metrics during Validation makes it easy to construct a dashboard create a new data series based on observed
```

Comment:
"a dashboard create" -> "a dashboard that creates" ?

Reply:
Fixed!